snipeit.sh

```diff
@@ -916,8 +956,8 @@ echo "  ***Open http://$fqdn to login to Snipe-IT.***"
 echo ""
 echo ""
 echo "* Cleaning up..."
-rm -f snipeit.sh
-rm -f install.sh
+# rm -f snipeit.sh
```

Comment:
This makes sense to me - snipeit.sh is part of the repo; no need for us to delete it, right?

Reply:
(and the same for install.sh)

Reply:
yeah. @snipe added that deletion back in the day. I pulled it because it was annoying to have to keep restoring it when testing, but I'm also not sure of the intent around deleting it.

Snipe, any input here?

Reply:
I can't imagine why I'd have done that - I didn't work much on this script at all (and largely loathe it's existence)

Reply:
good enough for me :-D thanks!